config/keptn/utils/teams.yaml

```diff
@@ -0,0 +1,14 @@
+repos:
+  - gh-action-extract-branch-name
+  - keptn-on-keptn
```

Comment:
this repo can be deleted. it would have been under the ops team, but it's not needed anymore